<launch>


  <include file="$(find jaco_driver)/launch/jaco_arm.launch">
  </include> 
<node name="aero_arm_controller" pkg="aero_control" type="arm_controller" output="screen">

  </node>
  <node name="aero_arm_velocity_controller" pkg="aero_control" type="arm_velocity_control" output="screen">


  </node>

</launch>
